script.js

```diff
@@ -1,4 +1,79 @@
+var currentGameMode = "waiting for user name";
+var userName = "";
+var playerWinCounter = 0;
+var botWinCounter = 0;
+
+// // Create a basic version of Scissors Paper Stone where the user inputs one of "scissors", "paper", or "stone", the program internally randomly chooses scissors, paper, or stone, and the program outputs whether the user won, the program won, or it's a draw.
+var scissors = "scissors";
+var paper = "paper";
+var stone = "stone";
+//randomly chooses scissors, paper, or stone
+var generateRandomHands = function () {
+  var randomNumber = Math.floor(Math.random() * 3) + 1;
+  // 1 = scissors
+  if (randomNumber == 1) {
+    return scissors;
+  }
+  // 2 = paper
+  if (randomNumber == 2) {
+    return paper;
+  }
+  // 3 = stone
+  return stone;
+};
+
+var gameStart = function (userName, userGuess) {
+  // Rules: scissors beats paper, paper beats stone, and stone beats scissors. If both parties choose the same object, it's a draw.
+  var robotHands = generateRandomHands();
+  var gameMessage = "";
+  // outputs whether the user won, the program won, or it's a draw.
+  var userWin = `${userName} choose ${userGuess}, <br> Bot choose ${robotHands}, <br> You Won! <br><br> Total Wins: <br> ${userName} VS Bot <br> ${playerWinCounter}:${botWinCounter}`;
+  var userLose = `${userName} choose ${userGuess}, <br> Bot choose ${robotHands}, <br> You Lose! Hahaha <br><br> Total Wins: <br> ${userName} VS Bot <br> ${playerWinCounter}:${botWinCounter}`;
+  var draw = `${userName} choose ${userGuess}, <br> Bot choose ${robotHands}, <br> It's a draw! <br><br> Total Wins: <br> ${userName} VS Bot <br> ${playerWinCounter}:${botWinCounter}`;
```

Comment:
Great work separating your message outputs. might be good to put this in a separate function 
E.g. 

```
var outputMessage = function (messageType) {
    if (messageType == 'userWin') {
        return ...
    }
    if (messageType == 'userLose') {
        return ...
    }
    if (messageType == 'draw') {
        return ...
    }
} 

```

// use this when needed 
`outputMessage ('userWin')`

Reply:
Good idea!